src/config.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Redis Ltd. 2016 - present
+ * Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or
+ * the Server Side Public License v1 (SSPLv1).
+ */
+
+#pragma once
+
+#include "redismodule.h"
+
+typedef struct {
+    long long value;
+    long long min;
+    long long max;
+} RM_ConfigNumeric;
+
+typedef struct {
+    double value;
+    double min;
+    double max;
+} RM_ConfigFloat;
+
+/*
+ * RM_Config is a global configuration struct for the module, it can be included from each file,
+ * and is initialized with user config options during module startup
+ */
+typedef struct {
+    /*********************************
+     * BLOOM FILTER CONFIG OPTIONS:  *
+     *********************************/
+    // Error ratio. valid range is [0.0, 1.0]. (default: 0.01)
+    // TODO: consider limiting the range to [0.0, 0.25] as per CONFIG PRD
+    RM_ConfigFloat bf_error_rate;
+    // Initial capacity. valid range is [1, 1e9] (default: 100)
+    RM_ConfigNumeric bf_initial_size;
+    // Expansion factor. valid range is [0, 32768] (default: 2)
+    // 0 is equivalent to NONSCALING
+    RM_ConfigNumeric bf_expansion_factor;
```

Comment:
I believe, there is almost no reason for this comments to exists here, when we already have a well-defined and easily readable table of the default values and the ranges for each of these.  I'd delete the comments explaining the valid ranges and so on, and leave only the comments explaining what it is etc.